serverFiles/models/comments.models.js

```diff
@@ -25,4 +25,33 @@ function addCommentByArticleId(article_id, commentData) {
     });
 }
 
-module.exports = addCommentByArticleId;
+function removeCommentById(comment_id) {
+  const regexPattern = /\d/g;
+  const isNumber = regexPattern.test(comment_id);
+  if (!isNumber) {
+    return Promise.reject({ status: 400, msg: 'Bad request' });
+  }
+
+  return db
+    .query(`DELETE FROM comments WHERE comment_id = $1 RETURNING *`, [
+      comment_id,
+    ])
+    .then(({ rows }) => {
+      if (!rows.length) {
+        return checkCommentExists(comment_id);
```

Comment:
I'm not sure whether this function is required, if the comment exists this should be deleted when the query is ran.